cmd/corald/main.go

```diff
@@ -48,7 +48,7 @@ func main() {
 		writeTimeout = 30 * time.Second
 	)
 
-	if err := app.Run(":4001", routes.API(), readTimeout, writeTimeout); err != nil {
+	if err := app.Run(":3000", routes.API(), readTimeout, writeTimeout); err != nil {
```

Comment:
I prefer to have all ports driven by env variables with defaults. This will allow us to orchestrate environments more flexibly when we move to deploy.